src/index.js

```diff
@@ -37,12 +39,42 @@ const initialCards = [
   },
 ];
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "e652845a-7220-4a96-9ecf-debf34d711a9",
+    "Content-Type": "application/json",
+  },
+});
+
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    cards.forEach((item) => {
+      const cardElement = getCardElement(item);
+      cardList.append(cardElement);
+    });
+    //Handle the user's information
+    profileImage.src = userInfo.avatar;
+    profileName.textContent = userInfo.name;
+    profileDescription.textContent = userInfo.about;
+  })
+  .catch(console.error);
+const submitButtonSelector = document.querySelectorAll(
```

Comment:
If you find DOM elements with `querySelectorAll`, you get a list of `DOM` elements. So you need to name the variable in **plural** (there is such a point in the checklist)


And it's not a selector. They are buttons


`submitButtons`